apps/aevm/src/aevm_eeevm.erl

```diff
@@ -1526,6 +1526,9 @@ recursive_call2(Op, Gascap, To, Value, OSize, OOffset, I, State8, GasAfterSpend)
                     {error, ?AEVM_PRIMOP_ERR_REASON_OOG(_OogResource, _OogCost, State9)} ->
                         ?TEST_LOG("Out of gas spending ~p gas for ~p", [_OogCost, _OogResource]),
                         eval_error(out_of_gas, State9);
+                    {error, not_allowed_off_chain} ->
+                        ?TEST_LOG("Not allowed calling this contract off-chain", []),
+                        eval_error(not_allowed_off_chain, State8);
```

Comment:
I see high level documentation in [state channels off chain doc - section on chain enforcement](https://github.com/aeternity/protocol/blame/26ba07353b62d236c3b9724f990479b8b4999e79/channels/OFF-CHAIN.md#L1041-L1042). Ok as high level but it feels a more detailed description would be beneficial. Please find proposal below.

It feels the not allowed primops shall be listed in https://github.com/aeternity/protocol/blame/26ba07353b62d236c3b9724f990479b8b4999e79/contracts/aevm.md#L45 like

> Calling [these VM primops] [in this situation] results in an error with output data `not_allowed_off_chain`.

Reply:
Addressed [Describe result of off-chain calls](https://github.com/aeternity/protocol/pull/235)